tests/integration/cases/micronaut-projects_micronaut-test/micronaut-test.dl

```diff
@@ -7,9 +7,9 @@ Policy("test_policy", component_id, "") :-
     check_passed(component_id, "mcn_build_as_code_1"),
     check_passed(component_id, "mcn_build_script_1"),
     check_passed(component_id, "mcn_build_service_1"),
-    check_passed(component_id, "mcn_provenance_available_1"),
-    check_passed(component_id, "mcn_provenance_level_three_1"),
     check_passed(component_id, "mcn_version_control_system_1"),
+    check_failed(component_id, "mcn_provenance_available_1"),
```

Comment:
Is it expected for these checks to fail due to the latest refactoring?
I wonder what was the reason for those checks to fail.

Reply:
This comment also applies to these test cases where checks turns from pass to fail
- `tests/integration/cases/slsa-framework_slsa-verifier/policy.dl`
- `tests/integration/cases/urllib3_expectation_dir/policy.dl`
- `tests/integration/cases/urllib3_expectation_file/policy.dl`
- `tests/integration/cases/urllib3_invalid_expectation/policy.dl`

Reply:
It is expected due to using the actual release instead of the latest. The commits provided for those test cases do not map to tags, and therefore there is no release to retrieve for any of them. 

We could change the commits to ones that have tags.

Reply:
Please fix the test case by providing PURL as input instead of commit  so that `mcn_provenance_available_1` passes when a provenance exists. You can also add an additional test case to assert `check_failed(component_id, "mcn_provenance_available_1"),` when the provided commit does not match a release tag.